### DISCUSSION TITLE
Aug 16 show and tell notes

### DISCUSSION THREAD

**Discussion by isabela-pf** [Quansight/Quansight.github.io#39] [Show and tell]
Aug 16 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

Please do not disclose confidential customer information.

**Comment by tonyfast** [comment]
# agenda

- duckly by @jaimergp and @ForgottenProgramme

**Reply by tonyfast** [reply]
[winapps](https://github.com/Fmstrat/winapps) by @costrouc

**Reply by isabela-pf** [reply]
More winapps links: 
- https://en.wikipedia.org/wiki/Remote_Desktop_Services#RemoteApp.
- https://forum.level1techs.com/t/how-to-seamlessly-run-windows-10-apps-in-a-vm-in-linux/170417

**Comment by isabela-pf** [comment]
[closember](https://closember.org/) by @carreau
inflatable kayaks do indeed work by @dharhas

**Reply by tonyfast** [reply]
https://github.com/dlenski/vpn-slice

**Reply by isabela-pf** [reply]
[jupyterlab-fonts](https://github.com/deathbeds/jupyterlab-fonts) by @peytondmurray on behalf of @tonyfast

**Reply by isabela-pf** [reply]
[meeseeksbox bot](https://meeseeksbox.github.io/) by @carreau

**Reply by tonyfast** [reply]
overriding css defaults with fonts https://github.com/deathbeds/hourhaus/pull/5#issuecomment-899562516

discussion about dyslexic fonts https://github.com/deathbeds/hourhaus/issues/4
